index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Luis's Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <nav class="flex-nav">
+        <ul>
+          <li class="logo">My Logo </li>
```

Comment:
- [ ] Your logo tag should be wrapped in an `a` tag as it is supposed to be a link and it should be aligned properly to the left to follow the given design.